playground/app.vue

```diff
@@ -46,10 +50,7 @@ onMounted(() => {
 <template>
   <section>
     <van-sticky>
-      <van-notice-bar
-        left-icon="volume-o"
-        text="无论我们能活多久，我们能够享受的只有无法分割的此刻，此外别无其他。"
-      />
+      <van-notice-bar left-icon="volume-o" text="无论我们能活多久，我们能够享受的只有无法分割的此刻，此外别无其他。" />
```

Comment:
Can you avoid formatting changes like this?

Reply:
This should be correcte in my latest commit.